types/nationalAvalancheCenter/observations.ts

```diff
@@ -0,0 +1,244 @@
+const Unknown = 'Unknown';
+
+function reverseLookup<T extends PropertyKey, U extends PropertyKey>(mapping: Record<T, U>, lookup: U): string {
+  return Object.entries(mapping)
+    .filter(([_, value]) => value === lookup)
+    .map(([key, _]) => key)
+    .reduce((accumulator, currentValue) => currentValue, Unknown);
+}
+
+export const Activity = {
+  'Skiing/Snowboarding': 'skiing_snowboarding',
+  'Snowmobiling/Snowbiking': 'snowmobiling_snowbiking',
+  'XC Skiing/Snowshoeing': 'xcskiing_snowshoeing',
+  Climbing: 'climbing',
+  'Walking/Hiking': 'walking',
+  Driving: 'driving',
+  Other: 'other',
+} as const;
```

Comment:
curious why you'd use the human-readable strings as keys and the api strings as values, instead of vice-versa. the reverse lookup seems like what you'd ~always want. 

or am I misunderstanding and the API is actually returning the human-readable strings? in which case WTF 😄

Reply:
Yeah, would love your input here - so for the mapping/display, I 100% agree that what you always want is the reverse mapping .. but when we're consuming data from the API, we want the value of the type to be e.g. `other` or `driving` (from the code you commented on) .... so it's a bit diametrically opposed.